CHANGES.md

```diff
@@ -4,7 +4,7 @@
 
 * Added 16 new locales for locale service and email templates (af, ar, bn, cz, hu, hy, jv, lt, no, ru, si, sv, ta, vi, zh-cn, zh-tw)
 * New users service routes to allow updates pref and name update
-* 2 stage Docker build
+* 2 stages Docker build
```

Comment:
2 stage build is the name of the feature.

Reply:
sure

Reply:
have changed